FunctionalTableData/TableView/TableCell.swift

```diff
@@ -28,6 +28,33 @@ public class TableCell<ViewType: UIView, Layout: TableItemLayout>: UITableViewCe
 		prepare?(view)
 		prepare = nil
 	}
+	
+	public override func accessibilityElementCount() -> Int {
```

Comment:
How come this is new? 🤔 I don't remember what we did to this

Reply:
I think we deliberately omitted this fuckery when we ported stuff back to the FTD repo last time. Let's leave it in otherwise we'll go insane.